crates/npmrc/src/custom_deserializer.rs

```diff
@@ -16,30 +16,40 @@ pub fn default_public_hoist_pattern() -> Vec<String> {
     vec!["*eslint*".to_string(), "*prettier*".to_string()]
 }
 
+/// Get absolute path with home_dir
+pub fn get_absolute_path_with_home_dir(relate_path: Option<String>) -> PathBuf {
```

Comment:
I don't think we need a function. We can just move this to `default_store_dir()`